server/devdocs/index.js

```diff
@@ -1,23 +1,25 @@
 /**
- * Extenal dependencies
+ * External dependencies
  */
-const express = require( 'express' ),
-	fs = require( 'fs' ),
-	fspath = require( 'path' ),
-	marked = require( 'marked' ),
-	lunr = require( 'lunr' ),
-	find = require( 'lodash/find' ),
-	escapeHTML = require( 'lodash/escape' );
+
+import express from 'express';
+import fs from 'fs';
+import fspath from 'path';
+import marked from 'marked';
+import lunr from 'lunr';
+import find from 'lodash/find';
```

Comment:
In Calypso we use the`babel-plugin-lodash` transform that does this for us, so we can also write:
```javascript
import { find, escape as escapeHTML } from 'lodash';
```